### PR TITLE
Avoid redundant code for refreshing Image trefc in imagelist

### PR DESCRIPTION
**Pull request by t2b3**
Also remove some empty lines

### CODE REVIEW

File: src/document.h [hunk 1]
```diff
@@ -724,6 +723,11 @@ struct Document {
         return GetBitmap().GetSubBitmap(r);
     }
 
+    void RefreshImageRefCount(bool includefolded = true) {
```
Comment (aardappel) [inline src/document.h:726]: Don't use default arguments unless on very frequently called functions or very exceptional arguments. Here, an explicit argument is better, it alerts the reader that a decision is being made.

File: src/grid.h [hunk 2]
```diff
@@ -334,7 +334,7 @@ struct Grid {
     }
 
     Selection SelectAll() { return Selection(this, 0, 0, xs, ys); }
-    void ImageRefCount() { foreachcell(c) c->ImageRefCount(); }
+    void ImageRefCount(bool includefolded) { if(includefolded || !folded) foreachcell(c) c->ImageRefCount(includefolded); }
```
Comment (aardappel) [inline src/grid.h:337]: good to spread more complex functions over multiple lines

File: src/document.h (hunk 1 above)
Comment (t2b3) [reply src/document.h]: Ah thanks for the review and the good general advice.

File: src/grid.h (hunk 2 above)
Comment (t2b3) [reply src/grid.h]: Ok I did this - hopefully right

**Comment by aardappel** [comment]
Thanks :)

**Pull request merged**